Expensify/App
# Create a group chat info view with participants information.

### Details
Added a modal view to show the list of all the participants in the group chat.

### Fixed Issues
Fixes [GH_LINK_ISSUE_2105](https://github.com/Expensify/Expensify.cash/issues/2105)

### Tests

1. Go to a group chat page.
2. Click on the title bar with the participants' names.
3. A modal with the participants' information appears.
4. On clicking each of the participants, the details page with each participant's information is displayed.

### Tested On


### Screenshots

#### Web
![image](https://user-images.githubusercontent.com/25154948/113201945-dfeda380-9289-11eb-82a3-97e2f3ee1221.png)
![image](https://user-images.githubusercontent.com/25154948/113201973-e8de7500-9289-11eb-8371-76c3ca04c990.png)


#### Mobile Web
![image](https://user-images.githubusercontent.com/25154948/113202029-fb58ae80-9289-11eb-8b44-782a09a22591.png)
![image](https://user-images.githubusercontent.com/25154948/113202091-10354200-928a-11eb-8a28-9ce549e01a6b.png)


#### Desktop
![image](https://user-images.githubusercontent.com/25154948/113202373-64d8bd00-928a-11eb-899f-c7e69e865035.png)
![image](https://user-images.githubusercontent.com/25154948/113202416-702be880-928a-11eb-93e5-91a5b2ece49c.png)


#### iOS
![aaaaaa](https://user-images.githubusercontent.com/25154948/113208565-ad47a900-9291-11eb-98de-0a9c5c051ea4.png)
![bbbbb](https://user-images.githubusercontent.com/25154948/113208575-af116c80-9291-11eb-8855-a7f7b5a847dc.png)


#### Android
![image](https://user-images.githubusercontent.com/25154948/113202776-eaf50380-928a-11eb-8a99-00e6b338a2ba.png)
![image](https://user-images.githubusercontent.com/25154948/113202811-f5170200-928a-11eb-884a-b90a3b96dc10.png)
